### DISCUSSION TITLE
Image | NanoPi R6C

### DISCUSSION THREAD

**Discussion by ghost** [MichaIng/DietPi#6312] [Image Request] [answered]
I have seen a comment in Source reference the NanoPi R6C; so I assume it's a consideration on that end.

Still: Pretty analogous to the R6S in some aspects.

Rockchip RK3588S
Up to 8GB RAM
Up to 32GB eMMC
30 Pin GPIO
Full-sized HDMI Port
Dual Ethernet Ports
USB-C UART
M.2 NVMe slot

Would be interesting to know if it has an ETA, or if anyone tried the R6S build on theirs.

**Comment by MichaIng** [comment]
I actually just received a sample and the NanoPi R6S image is just working well with it (same as NenoPi R5S and R5C are compatible). I just need to find a way to setup the Ethernet LEDs correctly, since on R6S the WAN adapter/LED is `eth2` which does not exist on R6C. So the LEDs won't show the cable connectivity correctly. However, minor thing.

**Comment by MichaIng** [comment, marked as answer]
Works very well: #6331
